app/assets/stylesheets/widgets.scss

```diff
@@ -365,3 +365,9 @@
     }
   }
 }
+
+.sponsorship-dropdown {
+  hr {
+    border: 1px solid var(--divider);
```

Comment:
As per mocks the divider height is 1px but because we are using `border` here, it considers the top and bottom border which makes it 2px.

<img width="259" alt="Screenshot 2023-04-12 at 11 30 46 PM" src="https://user-images.githubusercontent.com/9396084/231545401-658318c9-dbb3-4d2b-b481-e9428e2a1855.png">

Solution 1:
```
border: 0.5px solid var(--divider);
```

Solution 2:
```
border-top: 1px solid var(--divider);
border-bottom: 0px;
```

Reply:
Nice catch, thanks @rt4914!